pkg/manifest/local_server_test.go

```diff
@@ -0,0 +1,214 @@
+package manifest
+
+import (
+	"bytes"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/pelletier/go-toml"
+)
+
+type KVWrapper struct {
+	KVStores LocalKVStoreMap `toml:"kv_stores"`
+}
+
+func (w KVWrapper) MarshalTOML() ([]byte, error) {
```

Comment:
Same here.

Reply:
It turns out MarshalTOML wasn't part of some leftover code I had during development, so I removed it.